tests/core/secrets_collection_test.py

```diff
@@ -383,6 +413,12 @@ def test_load_baseline_without_exclude(self, mock_log):
             )
         assert mock_log.error_messages == 'Incorrectly formatted baseline!\n'
 
+    def get_point_thirteen_point_two_and_later_baseline_dict(self, gmtime):
+        # In v0.13.2 --from-commit got added
```

Comment:
++, great tests!

Reply:
thanks! I wanted to ask though, I added my additional assertion at [secrets_collection_test.py:390](https://github.com/Yelp/detect-secrets/pull/295/files/43120752a40f8602045adced247e8c4c0aaae21e#diff-1ec51425ce6e3b8ecdcb252ac8a6555eR390) trying to stay in style by duplicating the contents of the previous `test_load_baseline_from_string_with_X` methods; would it have been better to simply call those previous version tests in mine, and then test my assertions?